net/openssh/Makefile

```diff
@@ -211,12 +214,12 @@ define Build/Compile
 endef
 
 define Package/openssh-moduli/install
-	install -d -m0700 $(1)/etc/ssh
+	$(CREATE_SYSDIR)
```

Comment:
And this macro is better how than the previous row? I would just simply increase the chmod and thats it, where it is necessary to keep it understandable and simple.

Reply:
The other option is to have a macro for `0755` as `SYSDIR_PERMS` and use that.